doc/source/serve/tutorials/batch.md

```diff
@@ -112,9 +112,9 @@ results = ray.get([send_query.remote(text) for text in texts])
 print("Result returned:", results)
 ```
 
-You should get an output like the following. As you can see, the first batch has a 
-batch size of 1, and the subsequent queries have a batch size of 4. Even though each 
-query is issued independently, Ray Serve was able to evaluate them in batches.
+You should get an output like the following. The first batch has a 
+batch size of 1, and the subsequent queries have a batch size of 4. Even though Ray Serve issues each 
```

Comment:
```suggestion
batch size of 1, and the subsequent queries have a batch size of 4. Even though the client script issues each 
```

Reply:
TY! Is the client script the previous code snippet?